java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -380,6 +384,10 @@ private static Object[] sortedSetCommands(BaseTransaction<?> transaction) {
                     Map.of("one", 1.0, "two", 2.0), // zunionWithScores(KeyArray({zSetKey2, zSetKey1}));
                     Map.of("one", 1.0, "two", 2.0), // zunionWithScores(KeyArray({zSetKey2, zSetKey1}), MAX)
                     0L, // zinterstore(zSetKey1, new String[] {zSetKey2, zSetKey1})
+                    0L, // zinter(new KeyArray(new String[] {zSetKey2, zSetKey1}))
+                    0L, // zinter(new KeyArray(new String[] {zSetKey2, zSetKey1}), Aggregate.MAX)
+                    0L, // zinterWithScores(new KeyArray(new String[] {zSetKey2, zSetKey1}))
+                    0L, // zinterWithScores(new KeyArray(new String[] {zSetKey2, zSetKey1}), Aggregate.MAX)
```

Comment:
Isn't the return type of these transaction commands String[] for zinter and Map for zinterWithScores?

Reply:
Fixed, a copy-paste issue.